### ISSUE TITLE
The official application only works with VPN

### ISSUE THREAD

**Issue by Nataly-MR** [mastodon/mastodon-android#870]
The official Mastodon app since the last update does not load or work on my phone, it seems to be geographically blocked for Colombia, it only works by activating a VPN.

I use the Mastodon app version 2.6.1 obtained from GitHub, on a Samsung phone with Android 10, my account is hosted on the mastodon.social instance.
I have tried with other more updated phones, I have used both the GooglePlay and GitHub app but the same problem occurs; it does not load the feed, it does not update and there is no access to the account.

**Comment by grishka** [comment]
Are you able to access mastodon.social from a web browser?

**Comment by Nataly-MR** [comment]
> ¿Puedes acceder a mastodon.social desde un navegador web?

**Comment by Nataly-MR** [comment]
Yes, on the web I can connect without any problems, without the need of the VPN.

**Comment by jotaluissoy** [comment]
Please, I know what's been happening to Nataly. Mastodon web works perfectly to her, the issue is the app that is needing a VPN lately.
